spec/ameba/rule/lint/whitespace_around_macro_expression_spec.cr

```diff
@@ -0,0 +1,52 @@
+require "../../../spec_helper"
+
+module Ameba::Rule::Lint
+  describe WhitespaceAroundMacroExpression do
+    subject = WhitespaceAroundMacroExpression.new
+
+    it "passes if macro expression is wrapped with whitespace" do
```

Comment:
`{% %}` that aren't macro if / begin / verbatim / etc are also `MacroExpression`, there should be specs for those. Stuff like:

```crystal
{% puts "foo" %}
{%puts "foo" %}
{% puts "foo"%}
{%puts "foo"%}
```

Reply:
These are not being touched.

Reply:
Also, `{% ... %}` expressions are being formatted by the Crystal formatter anyway.